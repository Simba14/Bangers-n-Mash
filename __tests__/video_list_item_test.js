import React from 'react';
import VideoListItem from '../src/components/video_list_item';
import renderer from 'react-test-renderer';

test('renders the VideoListItem component correctly', () => {
  const props = {
    video:
      {
        default: {
          url: 'test'
        }
      }
  };

  const tree = renderer.create(
    <VideoListItem {...props}/>
  ).toJSON();
  expect(tree).toMatchSnapshot()
});
