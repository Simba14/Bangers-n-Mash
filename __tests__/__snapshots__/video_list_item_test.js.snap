// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the VideoListItem component correctly 1`] = `
<li
  className="list-items"
  onClick={[Function]}
>
  <div
    className="set-list-details"
  >
    <div
      className="details-left"
    >
      <img
        className="thumbnail"
        src="test"
      />
    </div>
    <div
      className="details-body"
    >
      <div
        className="details-heading"
      />
    </div>
  </div>
</li>
`;
